modules/src/ics04_channel/error.rs

```diff
@@ -44,8 +46,23 @@ pub enum Kind {
     #[error("missing counterparty")]
     MissingCounterparty,
 
+    #[error("no commong version")]
+    NoCommonVersion,
+
     #[error("missing channel end")]
     MissingChannel,
+
+    #[error("channel exists (was initialized) already: {0}")]
+    ChannelExistsAlready(ChannelId),
```

Comment:
This error msg is never used.

Reply:
Yes, it's from before the channel Id disappeared from the message. It was the error checkin the channel id in the ics04 spec that now is out of date.